tests/testthat/helper-data.R

```diff
@@ -0,0 +1,26 @@
+#' `h_get_data`
+#'
+#' @description `r lifecycle::badge("stable")`
+#'
+#' This is the helper function which creates a [`Data-class`] object
+#' for an example data.
+#'
+#' @return The [`Data-class`] object.
+#'
+h_get_data <- function() {
+  plcb <- 0.001
+  x <- c(plcb, 25, 25, 25, plcb, 50, 50, 50, plcb, 100, 100, 100)
+  dose_grid <- c(plcb, seq(25, 300, 25))
+
+  data <- .Data(
```

Comment:
```suggestion
  .Data(
```
avoid assigning to variable when it is the last value in function body